enhancements/installer/openstack-availibility-zones.md

```diff
@@ -0,0 +1,125 @@
+---
+title: openstack-availibility-zones
+authors:
+  - "@egarcia"
+  - "@pprenetti"
+reviewers:
+  - "@mandre"
+  - "@fedosin"
+approvers:
+creation-date: 2020-07-15
+last-updated: 2020-12-15
+status: implemented
+---
+
+# OpenStack Availability Zones
+
+## Release Signoff Checklist
+
+- [x]  Enhancement is `implementable`
+- [x] Design details are appropriately documented from clear requirements
+- [x] Test plan is defined
+- [x] Graduation criteria for dev preview, tech preview, GA
+- [x] User-facing documentation is created in [openshift-docs](https://github.com/openshift/openshift-docs/)
+
+## Open Questions
+
+- Will the node's AZ be used for its volume? Will this effect the behavior of the cluster?
+
+## Summary
+
+The installer should automatically discover all availability zones (AZs) and distribute the control plane and compute nodes across them to maximize availibility by default. To accomodate use cases with AZ restrictions, the installer should also allow users to specify a set of them for each machinepool.
+
+## Motivation
+
+### Goals
+
+- Enable the discovery of AZs in OpenStack
+- Spread out control plane and machine nodes across AZs to maximize availibility as a default behavior
+- Add an option to restrict a machine pool to a set of AZs in the installer
+
+### Non-Goals
+
+### Proposal
+
+### User Stories
+
+#### Day 2 Additional MachineSet
+
+As a user, I want to be able to add Machinepools on a different AZ then the one the installer is currently installed on in order to increase the availibility of my cluster, and to re-distribute the load.
+
+#### Install time Machinepool AZ customization
+
+As a user, I want to customize which AZs each MachineSet can be installed onto when the cluster is first installed.
```

Comment:
I want to customize which AZs each MachineSet can be installed onto

->

I want control over which Availability zones the Compute nodes are distributed to